components/Footer.css

```diff
@@ -4,3 +4,20 @@
     border-top: 2px solid var(--neutral-950);
     padding-top: 1rem;
 }
+
+.links {
+    margin-left: auto;
+    display: flex;
+    flex-wrap: wrap;
+    align-items: end;
```

Comment:
**issue/question:** You probably shouldn't need `margin-left: auto` here if aligning to the end anyway?